src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,206 @@
+package duke;
+
+import duke.task.*;
```

Comment:
Should the imported classes be listed explicitly?

Reply:
Hi, i used * as in this case, i needed all of the classes within the duke.task package, which were Task, ToDo, Deadline and Events classes to handle storage of different Task objects.